Sources/WAU/Winget-AutoUpdate/Winget-Install.ps1

```diff
@@ -168,8 +172,14 @@ function Install-App ($AppID, $AppArgs) {
             }
 
             #Add mods if deployed from Winget-Install
-            if ((Test-Path "$PSScriptRoot\mods\$AppID-preinstall.ps1") -or (Test-Path "$PSScriptRoot\mods\$AppID-upgrade.ps1") -or (Test-Path "$PSScriptRoot\mods\$AppID-install.ps1") -or (Test-Path "$PSScriptRoot\mods\$AppID-installed.ps1")) {
-                Add-WAUMods $AppID
+            if (Test-Path ".\mods\$AppID-*") {
+                #Check if WAU default install path exists
+                $Mods = "$WAUInstallLocation\mods"
+                if (Test-Path $Mods) {
+                    #Add mods
+                    Write-ToLog "-> Add modifications for $AppID to WAU 'mods'"
+                    Copy-Item ".\mods\$AppID-*" -Destination "$Mods" -Exclude "*installed-once*", "*uninstall*" -Force
+                }
```

Comment:
@KnifMelti could you remind me why you exclude `*uninstall*` here?

Reply:
I don't remember

Reply:
@Romanitho 
Because from `Winget-Install.ps1` it's the only time an **uninstallation** occurs, and that's a separate Repo.
WAU _only_ handles Mods for `-preinstall`, `-upgrade`, `-install`, `-installed` or `-notinstalled` itself.

Or, at least, that's what my memory tells me...

Reply:
My own way of using Mods in our production environment is to have centralized management (Web server) handling **all** Mods, nothing at all in `Winget-Install.ps1` **source**

Reply:
Ok thanks